### DISCUSSION TITLE
About the user

### DISCUSSION THREAD

**Discussion by nihalgurjar47** [dotneet/smart-chatbot-ui#56] [Q&A] [answered]
@dotneet  I want to just know how  can create multiple user.
My requirement is only that one user can't access another user's chat history, this features are available in this project or not?

****Feature Request****
I want to user management functionality,means if user exist in database than user can use **smart-chatbot-ui**  and if user does't exist in database than user can't use **smart-chatbot-ui**.

**Comment by dotneet** [comment, marked as answer]
@nihalgurjar47 

Thank you for your suggestion.

That feature should be really useful I think too.
But it can be achived by using `NEXTAUTH_EMAIL_PATTERN` currently.

For example, if you want to permit `john` and `mary` to use smart-chatbot-ui.
You can set NEXTAUTH_EMAIL_PATTERN like this:

```
NEXTAUTH_EMAIL_PATTERN=(john|mary)@example\.com
```
